web_src/css/base.css

```diff
@@ -630,6 +630,11 @@ a.label,
   color: var(--color-text-light-2);
 }
 
+.ui.dropdown > .text > .description,
+.ui.dropdown .menu > .item > .description {
+  color: var(--color-text-light-2);
+}
+
```

Comment:
This is not strictly for this PR anymore, but I saw we were missing this color for dark theme, so it's good to have it in any case.